nixos/modules/virtualisation/oci-options.nix

```diff
@@ -9,6 +9,13 @@
           Whether the OCI instance is using EFI.
         '';
       };
+      diskSize = lib.mkOption {
+        type = lib.types.int;
+        default = 8192;
+        description = ''
+          Size of the disk image created.
```

Comment:
Would be good to mention the unit used. And an example that a human can properly understand.

Reply:
Good point. Respun.